src/hosted-buttons/utils.test.js

```diff
@@ -89,6 +93,107 @@ test("getHostedButtonDetails", async () => {
       shape: "rect",
       color: "gold",
       label: "paypal",
+      tagline: true,
+    });
+  });
+  expect.assertions(1);
+});
+
+test("getHostedButtonDetails handles string values for tagline", async () => {
+  // $FlowIssue
+  request.mockImplementationOnce(() =>
+    // eslint-disable-next-line compat/compat
+    Promise.resolve({
+      body: {
+        button_details: {
+          link_variables: [
+            {
+              name: "business",
+              value: merchantId,
+            },
+            {
+              name: "shape",
+              value: "rect",
+            },
+            {
+              name: "layout",
+              value: "vertical",
+            },
+            {
+              name: "color",
+              value: "gold",
+            },
+            {
+              name: "button_text",
+              value: "paypal",
+            },
+            {
+              name: "tagline",
+              value: "false",
+            },
+          ],
+        },
+      },
+    })
+  );
+  await getHostedButtonDetails({
+    hostedButtonId,
+    fundingSources: [],
+  }).then(({ style }) => {
+    expect(style).toEqual({
+      layout: "vertical",
+      shape: "rect",
+      color: "gold",
+      label: "paypal",
+      tagline: false,
+    });
+  });
+  expect.assertions(1);
+});
+
+test("getHostedButtonDetails handles missing value for tagline", async () => {
+  // $FlowIssue
+  request.mockImplementationOnce(() =>
+    // eslint-disable-next-line compat/compat
+    Promise.resolve({
+      body: {
+        button_details: {
+          link_variables: [
+            {
+              name: "business",
+              value: merchantId,
+            },
+            {
+              name: "shape",
+              value: "rect",
+            },
+            {
+              name: "layout",
+              value: "vertical",
+            },
+            {
+              name: "color",
+              value: "gold",
+            },
+            {
+              name: "button_text",
+              value: "paypal",
+            },
+          ],
+        },
+      },
+    })
+  );
+  await getHostedButtonDetails({
+    hostedButtonId,
+    fundingSources: [],
```

Comment:
I think `fundingSources` can be removed here

Reply:
Yeah, I would remove these. This is how we are temporarily rendering the v2 experience but will remove the `fundingSources` in this [ticket](https://paypal.atlassian.net/browse/DTPPCPSDK-2229) that's currently in the sprint.

The tagline only gets rendered in the horizontal smart stack and the v2 experience is rendering 2 standalone buttons.

Reply:
It's currently required on the type. Are we removing it completely in that ticket? Might make it optional for now, and @nbierdeman can address in that ticket?